src/react/Orb.js

```diff
@@ -0,0 +1,281 @@
+﻿"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["src/common/HTMLWidget","../common/Utility","../common/PropertyExt","css!orb", "css!./Orb"], factory);
+    } else {
+        root.template_Orb = factory(root.common_HTMLWidget, root.common_Utility, root.common_PropertyExt, root.React);
+    }
+}(this, function (HTMLWidget, Utility, PropertyExt) {
+    var orb = null
+    function Orb(target) {
+        HTMLWidget.call(this);
+    }
+
+
+function mapping(owner){
+	PropertyExt.call(this);
+	this._owner = owner;
+}
+
+
+var fields =[];
+var savedField = [];
+var savedTableField = [];
+var rowFields = [];
+var dataFields = [];
+var columnFields = [];
```

Comment:
These are effectively global variables for all instances of the Orb class - move them into the Orb class constructor:
```
function Orb(target) {
    HTMLWidget.call(this);
    this.fields =[];
    this.savedField = [];
    etc.
}
```